javascript/javascript1/week2/task_1.js

```diff
@@ -0,0 +1,52 @@
+//function to get fullname of a person
+
+function getFullname(firstname, lastname) {
+  const fullname = firstname + " " + lastname;
+  return fullname;
+}
+
+fullname1 = getFullname("Parul", "Jain");
+fullname2 = getFullname("Peter", "Pan");
+console.log(fullname1);
+console.log(fullname2);
+
+//code for - when argument for useFormalName is provided when function is called
+
+function getFullname(firstname, lastname, useFormalName) {
+  if (useFormalName == true) {
+    const fullname = "Lord" + " " + firstname + " " + lastname;
+
+    return fullname;
+  } else {
+    const fullname = firstname + " " + lastname;
+
+    return fullname;
+  }
+}
+
+getFullname("", "");
+
+/* function if there is a checkbox for gender of the user and user formal name*/
+
+function getFullname(firstname, lastname, useFormalName, gender) {
```

Comment:
Don't define many functions with the same name. The last one will override the previous function.
Merge them all in one including all requirements.